contracts/test/contract/arbRollup.spec.ts

```diff
@@ -223,6 +223,7 @@ const setup = async () => {
 
 async function tryAdvanceChain(blocks: number): Promise<void> {
   try {
+    await ethers.provider.send("evm_increaseTime", [blocks * 12]);
```

Comment:
let's add a test case that expect revert if only half time passed i.e. `blocks * 12 / 2` ?

Reply:
I realized that the way we did timing in the tests was pretty broken (e.g. instead of advancing to block X we'd advance X blocks). I think I've fixed all the timing in the tests, and added an assertion that the challenge doesn't timeout early.